frontend_tests/casper_tests/14-drafts.js

```diff
@@ -43,6 +43,15 @@ casper.then(function () {
     waitWhileDraftsVisible();
 });
 
+casper.then(function () {
+    casper.test.info('Opening Markdown Preview');
+    casper.click('body');
+    casper.page.sendEvent('keypress', "c");
+    casper.waitUntilVisible('#markdown_preview', function () {
+        casper.click('#markdown_preview');
+    });
+});
+
```

Comment:
So having this action here brings out an un-intended effect which is that pressing c or C to open the create a stream/private message doesn't close the preview which we don't want to depend on for this test to pass I guess? We should probably move this to just above the clicking on the drafts link.

Reply:
Oh and, the preview gets closed by the `casper.click("#compose_close");` that is run after creating the drafts, so it doesn't really do it's job :stuck_out_tongue: